pip/flatpak-pip-generator

```diff
@@ -228,6 +230,8 @@ with tempfile.TemporaryDirectory(prefix=tempdir_prefix) as tempdir:
         if not filename.endswith(('bz2', 'gz', 'xz', 'zip')):
             version = get_file_version(filename)
             name = get_package_name(filename)
+            if  opts.ignore_pkg and name in opts.ignore_pkg:
+                continue
```

Comment:
> Hrm, looks like for some reason packages I supposedly ignored on the commandline ended up in the final manifest, but using wheel sources? Not sure if that's a feature or a bug...

It's a feature. This condition is only applied during "Download arch independent packages"  stage. Prior to that wheels are downloaded at line 215/217. You will need to either interfere before that or at the manifest building stage